woocommerce/woocommerce-ios
# Remove the new orders card from V3 Stats

(Does not exist in V4 Stats)

Fixes #1295 
Closes #946 

Note: this one will need merged into 2.7's build

## To test
1. Have a store running v3 stats
2. Check out this branch and build
3. No New Orders card should appear

### Screenshots
<img src="https://user-images.githubusercontent.com/1062444/65529007-ff4c8900-deba-11e9-8edf-2af18d9289b6.png" width="350" />


Update release notes:
